yaml_files/simphony_metadata.yml

```diff
@@ -170,6 +170,7 @@ CUDS_KEYS:
     CUBA.RHEOLOGY_MODEL:
       default: CUBA.NEWTONIAN_FLUID_MODEL
     CUBA.GRAVITY_MODEL:
+      scope: CUBA.USER
```

Comment:
@tuopuu If we provide a value for `default` here, it will be possible to instantiate `Cfd` without passing any arguments.